### ISSUE TITLE
Code of Conduct Acceptance

### ISSUE THREAD

**Issue by cassar** [govhackaustralia/hackerspace3#41]
## Background

Currently there is no Code of Conduct to agree to in the hackerspace.

From Competition 2019, Competitors will need to agree to a code of conduct when they register for their first event.

For subsequent events a note will be displayed on the bottom to say that they already agreed to the code of conduct and a link to the text.

**Update**: The below work is on the branch *code-of-conduct*, work can be merged once #2 is complete.

## Tasks

- [x] Add a static page for the Code of Conduct test. 
- [x] Add an attribute to the user model, accepted_code_of_conduct, as a timestamp.
- [x] Add a validation on the registration model to not permit a registration unless the code of conduct has been agreed to.
- [x] Add a section to the admin users show page to indicate if a user has accepted the code of conduct.
- [x] Add an extra field on the registration form, a checkbox, to agree to the code of conduct.
- [x] Add an extra field on the waiting list form, a checkbox, to agree to the code of conduct.
- [x] Add a message to the registration form that the code of conduct has been agreed to and where the user can find it.
- [ ] Add ApplicationController level restriction on accepting code of conduct.

**Comment by cassar** [comment]
https://govhack.org/competition/code-of-conduct/

**Comment by cassar** [comment]
Closing this, 

To request code of conduct be agreed to a sign up (rather than event registration), please open a new ticket.
